packages/core/core/src/requests/BundleGraphRequest.js

```diff
@@ -376,6 +376,16 @@ class BundlerRunner {
         await this.runDevDepRequest(devDepRequest);
       }
     } catch (e) {
+      invariant(internalBundleGraph != null);
```

Comment:
A failing assertion here will overwrite the actual error that was caught, you should wrap the `storeResult` call in an `if`. Should behave the same but retain the error message